.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,40 @@
+name: "Run Tests"
+
+on:
+  pull_request:
+
+jobs:
+  run_tests:
+    name: "Run Tests on PHP ${{ matrix.php_version }}"
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        php_version: ['7.4', '8.0']
+        experimental: [false]
+        include:
+          - php_version: '7.2'
+            os: ubuntu-18.04
+            experimental: false
+          - php_version: '7.3'
+            os: ubuntu-18.04
+            experimental: false
+          - php_version: '8.1'
+            os: ubuntu-latest
+            experimental: true
```

Comment:
neat

Reply:
Yeah, I thought that was a cool feature, so if a nightly build happens to fail, or if they suddenly introduce something that throws a deprecation warning, we'll see it without it "breaking" the tests and we'll have time to address it.